src/views/swap/steps/sendTransaction.js

```diff
@@ -24,32 +26,48 @@ const SendTransactionStyles = () => ({
   info: {
     flexDirection: 'column',
     flex: 1,
-    justifyContent: 'space-around',
   },
   text: {
-    fontSize: '30px',
+    fontSize: '20px',
+    '@media (max-width: 425px)': {
+      fontSize: '16px',
+    },
   },
   address: {
     width: '300px',
-    fontSize: '25px',
+    fontSize: '18px',
     color: 'grey',
     wordBreak: 'break-word',
+    '@media (max-width: 425px)': {
+      fontSize: '16px',
+    },
   },
   action: {
     color: 'blue',
     fontWeight: '600',
-    fontSize: '30px',
-    marginLeft: '50%',
+    fontSize: '20px',
+    marginLeft: '60%',
     '&:hover': {
       cursor: 'pointer',
     },
+    '@media (max-width: 425px)': {
+      fontSize: '16px',
+    },
   },
 });
 
 const StyledSendTransaction = ({ classes, swapInfo, swapResponse }) => (
   <View className={classes.wrapper}>
     <View className={classes.qrcode}>
-      <QrCode size={300} link={swapResponse.bip21} />
+      <DetectResize>
+        {width =>
+          width <= 425 ? (
```

Comment:
Why are you using `425` instead of [`375`](https://github.com/BoltzExchange/boltz-frontend/pull/129/files#diff-9afcf9c655e03704e51698cc0b82bff9R87) here?

The `payInvoice` screen looks good on my phone now but this one has the same scaling issues as before you added the `DetectResize` compoenent.

Reply:
It worked better when I tested it. But removed it.